etc/sync-function-template.js

```diff
@@ -760,7 +767,15 @@ function synctos(doc, oldDoc) {
       var channels = resolveCollectionDefinition(doc, effectiveOldDoc, definition.channels);
 
       access(usersAndRoles, channels);
+
+      effectiveAssignments.push({
+        type: 'channel',
```

Comment:
This parameter has been included in preparation for the implementation of #61, which will make it possible to assign roles to users (i.e. its `type` will be "role").